tests/sentry/tasks/test_store.py

```diff
@@ -104,7 +92,7 @@ def mock_metrics_timing():
 def test_move_to_process_event(
     default_project, mock_process_event, mock_save_event, mock_symbolicate_event, register_plugin
 ):
-    register_plugin(BasicPreprocessorPlugin)
+    register_plugin(globals(), BasicPreprocessorPlugin)
```

Comment:
Not sure what's happening here

Reply:
This is just wonky test setup to register and unregister plugins. `register_plugin` used to be defined in this module (and just called `globals()` itself), but somehow I couldn't get it to work in conftest.py.

The reason we use globals is because the plugin manager needs a valid import-path to import the module from, so we temporarily provide that in tests.